app/style/components/group-video-grid.less

```diff
@@ -0,0 +1,66 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+.video-grid {
+  position: relative;
+  width: 100%;
+  height: 100vh;
+}
+
+.video-grid__thumbnail {
+  position: absolute;
+  top: 1em;
+  right: 1em;
+  width: 7em;
+  height: 6em;
+  border: solid 2px white;
+}
+.video-grid__element {
+  position: absolute;
+  width: 50%;
+  height: 50%;
+  border: solid 2px white;
+}
+.video-grid__element0 {
+  top: 0;
+  left: 0;
+}
+.video-grid__element1 {
+  bottom: 0;
+  left: 0;
+}
+.video-grid__element2 {
+  top: 0;
+  right: 0;
+}
+.video-grid__element3 {
+  right: 0;
+  bottom: 0;
+}
+.video-grid__element--full-height {
+  height: 100%;
+}
+.video-grid__element--full-size {
+  width: 100%;
+  height: 100%;
+}
+.video-grid__element--empty {
+  overflow: hidden;
+  width: 0;
+  height: 0;
```

Comment:
```less
.video-grid {
  position: relative;
  width: 100%;
  height: 100vh;
  
  &__thumbnail {
    position: absolute;
    top: 1em;
    ...
  }

  &__element {
    position: absolute;
    width: 50%;
  }
  ...
}
 
```

Reply:
Oh, nice trick. Didn't know this one :+1:

Reply:
Works with SCSS as well.. :)

Also have an empty line between every rule.. apparently Prettier is not doing it :(